codecs/codecs-btp/src/test/java/org/interledger/codecs/btp/AsnBtpErrorDataSerializationTest.java

```diff
@@ -147,10 +152,10 @@ public void read() throws Exception {
      */
     final BtpError actualValue = codecContext.read(BtpError.class, inputStream);
 
-    assertThat(actualValue.getRequestId(), is(this.expectedRequestId));
-    assertThat(actualValue.getErrorCode(), is(this.expectedErrorCode));
-    assertThat(actualValue.getTriggeredAt(), is(this.expectedTriggeredAt));
-    assertThat(actualValue.getErrorData(), is(BaseEncoding.base64().decode(this.expectedErrorDataBase64Bytes)));
+    MatcherAssert.assertThat(actualValue.getRequestId(), Matchers.is(this.expectedRequestId));
```

Comment:
Minor nit: Let's go back to just `assertThat(...`. In the case of unit tests, IMHO, using `MatcherAssert.assertThat` everywhere impedes readability without much gain.